core/services/ocr2/plugins/ocr2vrf/coordinator/coordinator.go

```diff
@@ -41,21 +43,39 @@ const (
 
 	// Both VRF and DKG contracts emit this, it's an OCR event.
 	configSetEvent = "ConfigSet"
+
+	// TODO: Add these defaults to the off-chain config, and get better gas estimates
+	// (these current values are very conservative).
```

Comment:
Worth a discussion re: how this looks like as a off-chain config vs. hard-coded in the code (on a per-chain basis).